frontend/src/components/Settings/settings/tabs/IntegrationsTab.tsx

```diff
@@ -51,49 +51,49 @@ const IntegrationsTab: FC = () => {
       }
-      nango
-        .auth(unique_config_key, item.connection_id)
-        .then((result) => {
-          toast.success(
-            `OAuth flow succeeded for provider "${result.providerConfigKey}"!`
-          );
-          console.log("RESULT FROM OAUTH: ", result);
-          const inner_data: BraintreeConnectionRequestType = {
-            nango_connected: true,
-          };
-          const request_data: PaymentProcessorConnectionRequestType = {
-            payment_processor: "braintree",
-            data: inner_data,
-          };
+      // nango
+      //   .auth(unique_config_key, item.connection_id)
+      //   .then((result) => {
+      //     toast.success(
+      //       `OAuth flow succeeded for provider "${result.providerConfigKey}"!`
+      //     );
+      //     console.log("RESULT FROM OAUTH: ", result);
+      //     const inner_data: BraintreeConnectionRequestType = {
+      //       nango_connected: true,
+      //     };
+      //     const request_data: PaymentProcessorConnectionRequestType = {
+      //       payment_processor: "braintree",
+      //       data: inner_data,
+      //     };
 
-          PaymentProcessorIntegration.connectPaymentProcessor(request_data)
-            .then((data: PaymentProcessorConnectionResponseType) => {
-              toast.success(data.details);
-            })
-            .catch((error) => {
-              toast.error(error.details);
-            });
-          refetch();
-        })
-        .catch((error) => {
-          const inner_data: BraintreeConnectionRequestType = {
-            nango_connected: true,
-          };
-          const request_data: PaymentProcessorConnectionRequestType = {
-            payment_processor: "braintree",
-            data: inner_data,
-          };
-          PaymentProcessorIntegration.connectPaymentProcessor(request_data)
-            .then((data: PaymentProcessorConnectionResponseType) => {
-              toast.success(data.details);
-              refetch();
-            })
-            .catch((inner_error) => {
-              toast.error(
-                `There was an error in the OAuth flow for integration: ${error.message}`
-              );
-            });
-        });
+      //     PaymentProcessorIntegration.connectPaymentProcessor(request_data)
+      //       .then((data: PaymentProcessorConnectionResponseType) => {
+      //         toast.success(data.details);
+      //       })
+      //       .catch((error) => {
+      //         toast.error(error.details);
+      //       });
+      //     refetch();
+      //   })
+      //   .catch((error) => {
+      //     const inner_data: BraintreeConnectionRequestType = {
+      //       nango_connected: true,
+      //     };
+      //     const request_data: PaymentProcessorConnectionRequestType = {
+      //       payment_processor: "braintree",
+      //       data: inner_data,
+      //     };
+      //     PaymentProcessorIntegration.connectPaymentProcessor(request_data)
```

Comment:
Pls watch out for removing nango

Reply:
@glamboyosa Yes please don't remove this.

Reply:
Just put it back